README.md

```diff
@@ -9,11 +9,13 @@ They provide common runtime utilities needed by SDKs to make API calls and handl
 | Name | Description |
 | --- | --- |
 | [@apimatic/schema](packages/schema) | Validate and transform data using schema definitions. |
+| [@apimatic/authentication-adapters](packages/authentication-adapter) | Provides authentication schemes pluggables |
+| [@apimatic/axios-client-adapter](packages/axios-client-adapter) | Provides http client pluggables. |
+| [@apimatic/convert-to-stream](packages/convert-to-stream) | Provides stream conversion support. |
 | [@apimatic/core](packages/core) | Provides core http logic, file wrapper, api response and validation error classes. |
-
-## License
-
-© 2020 APIMatic Ltd. All Rights Reserved.
-
-This library is meant to be used in SDKs generated by APIMatic Code Generator. Please get in
-touch with our [support](mailto:mehdi@apimatic.io) for more information on licensing.
+| [@apimatic/core-interfaces](packages/core-interfaces) | Provides common interfaces to core. |
+| [@apimatic/file-wrapper](packages/file-wrapper) | Provides file wrapper support to core. |
+| [@apimatic/http-headers](packages/http-headers) | Provides http hedaers support to core. |
+| [@apimatic/http-query](packages/http-query) | Provides http query support to core. |
+| [@apimatic/oauth-adapters](packages/oauth-adapters) | Provides oauth schemes pluggables. |
+| [@apimatic/xml-adapter](packages/xml-adapter) | Provides xml serialization and deserialization support to core. |
```

Comment:
Provides XML serialization and deserialization utilities for apimatic-js-runtime libraries.